issuance/issuance.go

```diff
@@ -598,16 +599,24 @@ type IssuanceRequest struct {
 	SCTList           []ct.SignedCertificateTimestamp
 }
 
-// Issue generates a certificate from the provided issuance request and
-// signs it. Before signing the certificate with the issuer's private
-// key, it is signed using a throwaway key so that it can be linted using
-// zlint. If the linting fails, an error is returned and the certificate
-// is not signed using the issuer's key.
-func (i *Issuer) Issue(req *IssuanceRequest) ([]byte, error) {
+// An issuanceToken represents an assertion that Issuer.Lint has generated
+// a linting certificate for a given input and run the linter over it with no
+// errors. It can be used to issue a real certificate (once).
```

Comment:
```suggestion
// errors. The token will be redeemed during issuance of one final certificate.
```

Reply:
Something like this.

Reply:
Good point that "real" certificate is not well defined. However, "final" certificate is also not quite right here either since this can be either a precertificate or a final certificate. I tried to make it clearer - let me know what you think!